src/toil/jobStores/abstractJobStore.py

```diff
@@ -93,6 +94,49 @@ def writeConfigToStore(self):
     def config(self):
         return self.__config
 
+    def setAsRootJob(self, rootJobStoreID):
+        """
+        Sets the root job for the current job store.
+
+        :param str rootJobStoreID: The id of the job to set as root
+        """
+        with self.writeSharedFileStream("rootJobStoreID") as f:  # Load the root job
+            f.write(rootJobStoreID)
+
+    def loadRootJob(self):
+        """
+        Loads the root job in the current job store.
+
+        :raises toil.job.JobException: If root job is not in the job store.
+        :return: The root job.
+        :rtype: toil.jobWrapper.JobWrapper
+        """
+        try:
+            with self.readSharedFileStream("rootJobStoreID") as f:  # Load the root job
+                rootJobID = f.read()
+        except NoSuchFileException:
+            raise RuntimeError("No job has been set as root for the current job store")
+
+        if not self.exists(rootJobID):
+            raise JobException("No root job (%s) left in toil workflow (workflow "
```

Comment:
The root job '%s' doesn't exist. Either the Toil workflow is finished or has never been started.